### PR TITLE
feat(docs): vale styles creation part 1

### PR DESCRIPTION
**Pull request by RoRoJ**
**What does this PR do?**
- Creates a new subfolder `scw_styles` to hold all style rules that we want Vale to use for our docs
- Creates new style `yml` files in this subfolder (some created from scratch, some adapted from Microsoft/write-good styles) which enforce rules from the Scaleway Documentation Guidelines (https://confluence.infra.online.net/display/MARKETTECHANIM/Documentation+Guidelines)
- Edits `vale.ini`to tell it to use only these styles/rules

- Creates new subfolder `scw_styles_indev` to hold style rules that are in development
- These styles are for rules that are harder to implement and require additional work/research
- Finishing these styles will be done in the future in a separate PR
- Vale is not currently applying styles in this subfolder

**Supporting document with more information on Vale and the style rules we have implemented**
See: https://docs.google.com/document/d/1ptiOAhmXlvrgc_N1iV2MoNaom05v8tingbsrfqpPm08/edit

**To test**
- pull this PR and navigate to the content folder or any of its subfolders
- run `vale` + `name-of-document` eg `vale compute/instances/how-to/create-an-instance.mdx`
- don't hesitate to test the rules by altering a sample document with "bad" wording to check that Vale picks up on it

### CODE REVIEW

File: styles/scw_styles_indev/EnvVars.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+extends: substitution
+message: "Use '%s' instead of '%s'."
+link: https://confluence.infra.online.net/display/MARKETTECHANIM/Documentation+Guidelines#DocumentationGuidelines-EnvironmentVariables
+level: warning
+
+ignorecase: true
+action:
+  name: replace
+swap:
+  $CALEWAY_ACCESS_KEY|$ACCESS_KEY: $SCW_ACCESS_KEY
```
Comment (tthsympa) [inline styles/scw_styles_indev/EnvVars.yml:10]: $CALEWAY_ACCESS_KEY -> $SCALEWAY_ACCESS_KEY

File: styles/scw_styles_indev/TitleFormat.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+extends: ?
```
Comment (tthsympa) [inline styles/scw_styles_indev/TitleFormat.yml:1]: What `extends: ?` does?

Comment (tthsympa) [reply styles/scw_styles_indev/TitleFormat.yml:1]: It means "do nothing for now" ?

File: styles/scw_styles_indev/EnvVars.yml (hunk 1 above)
Comment (RoRoJ) [reply styles/scw_styles_indev/EnvVars.yml:10]: merci, je pense que ca c'est parce que j'ai testé pleine de choses pour faire marcher cette regle mais je n'arrive pas. Je pense que les `$` derange Vale. J'ai essayé avec $$ et \$ mais dans tous les cases, ca ne declenche jamais des warnings/suggestions meme quand ca devrait se faire.

File: styles/scw_styles_indev/TitleFormat.yml (hunk 2 above)
Comment (RoRoJ) [reply styles/scw_styles_indev/TitleFormat.yml:1]: yeah, the rules in the `scw_styles_indev` folder are the ones that we need to create (placeholders) or ones that we've created but aren't working properly.

File: styles/scw_styles_indev/EnvVars.yml (hunk 1 above)
Comment (RoRoJ) [inline styles/scw_styles_indev/EnvVars.yml:10]:
```suggestion
  $SCALEWAY_ACCESS_KEY|$ACCESS_KEY: $SCW_ACCESS_KEY
```

Comment (tthsympa) [reply styles/scw_styles_indev/EnvVars.yml]: et avec "xxx" ?

**Pull request closed without merging**